src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const pricePerDay = 40;
+  const threeDaysStay = 3;
+  const sevenDaysStay = 7;
+
+  const threeOrMoreDaysDiscount = 20;
+  const sevenOrMoreDaysDiscount = 50;
+  const basicPrice = days * pricePerDay;
+  const lessThanThreeDaysCost = basicPrice;
+  const lessThanSevenDaysCost = days * pricePerDay - threeOrMoreDaysDiscount;
+  const sevenOrMoreDays = days * pricePerDay - sevenOrMoreDaysDiscount;
+
+  if (days < threeDaysStay) {
+    return lessThanThreeDaysCost;
+  } else if (days < sevenDaysStay) {
+    return lessThanSevenDaysCost;
+  }
```

Comment:
```suggestion
  if (days < threeDaysStay) {
    return lessThanThreeDaysCost;
  } 
  
  if (days < sevenDaysStay) {
    return lessThanSevenDaysCost;
  }
```

We don't need to use an else block, because in each if we have a return statement